In [1]:
import pandas as pd

In [2]:
df = pd.read_csv(r"C:\Users\vasil\OneDrive\ScamLens raw datasets\phishing emails.csv", encoding = 'latin-1')

In [3]:
df.head(8)


,label,text
0,1,ounce feather bowl hummingbird opec moment ala...
1,1,wulvob get your medircations online qnb ikud v...
2,0,computer connection from cnn com wednesday es...
3,1,university degree obtain a prosperous future m...
4,0,thanks for all your answers guys i know i shou...
5,0,larry king live at escapenumber escapenumber p...
6,0,michael pobega wrote i'm not sure if it's the ...
7,0,hi i have this error tr sample escapenumber es...


In [4]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 83448 entries, 0 to 83447
Data columns (total 2 columns):
 #   Column  Non-Null Count  Dtype 
---  ------  --------------  ----- 
 0   label   83448 non-null  int64 
 1   text    83448 non-null  object
dtypes: int64(1), object(1)
memory usage: 1.3+ MB


In [5]:
df.duplicated().sum()

np.int64(0)

In [6]:
df.describe()

,label
count,83448.000000
mean,0.526196
std,0.499316
min,0.000000
25%,0.000000
50%,1.000000
75%,1.000000
max,1.000000


#### This dataset contains 83448 rows. The dataset was checked for Null values and Duplications. There is no any duplications nad Null values. There are two columns one Label , which represents as 1 and 0 , 1 represents the Spam emails, while the 0 represents the Ham emails. Dataset is ready for saving into data folder in cleaned file for next phase of preparation of the files.

In [8]:
df.to_csv(r'C:\Users\vasil\OneDrive\ScamLens Dissertation\ScamLens Project\data\cleaned\email_cleaned.csv', index=False)